### ISSUE TITLE
how-to: Use differentADMOB_APP_ID for Android and iOS

### ISSUE THREAD

**Issue by svzi** [ratson/cordova-plugin-admob-free#302]
I tried to update this plugin to the current version, but I'm unable to add the ADMOB_APP_ID via CLI installation, because I use a different ADMOB_APP_ID for each mobile operating system (one for Android, one for iOS).

What am I supposed to do in order to get the newest version of this plugin running?

**Comment by zigou13** [comment]
I have the same concern as you, if we have two ADMOB IDs (one for IOS and one for Android)

**Comment by svzi** [comment]
@ratson Any update on this?

**Comment by fmendoza** [comment]
Maybe App ID is not required for iOS?

**Comment by svzi** [comment]
@fmendoza I don't know how that should help. Did you read my first post?

**Comment by lucaele** [comment]
Same request, any fix?

**Comment by svzi** [comment]
I've switched to https://github.com/admob-plus/admob-plus meanwhile.

**Comment by edento** [comment]
+1

**Comment by AhSem** [comment]
I have the same concern. When creating an App in Google Admob, we can choose the App platform to be Android or iOS. In most cases, user will create separate App ID for both platforms

This plugin seems to only able to add one App ID. In such case, banner ad is not showing in my Android device, but does appear in my iOS device. Not sure if this is because the APP ID is not matching the Ad Unit ID in my Android phone.

**Comment by AhSem** [comment]
@edento , thank you for your reply. Correct me if I am wrong. You mean that in Google Admob, regardless what platform we created the App ID, it can be used interchangeably for both Android and iOS platforms in this plugin?

**Comment by edento** [comment]
@AhSem Yes. Maybe it shouldn't, but it does. My app now has ads in both platforms with same ID, but make sure not to miss this: I am still on testing mode, maybe it wouldn't work on production.

**Comment by tbergeron** [comment]
Any update on this?

And yes you need to use separate APP IDs. Even Google tells to use different IDs for different OSes (i.e. you cannot create an admob app for both OSes)

So as of right now we have no answers from @ratson nor a way of handling a separate APP_ID per OS??? I guess I will have to use another plugin :-/

**Comment by srikanth-wgl** [comment]
Looking at the code it is clear that this plugin supports APP ID for Android only. AdMob SDK for Android requires APP ID in AndroidManifest file since version 17.0.0. This plugin satisfies that requirement.

@ratson has another plugin under development that supports APP IDs for Android and iOS. See https://github.com/admob-plus/admob-plus for more info.
